### DISCUSSION TITLE
Is there a way to prune the system log?

### DISCUSSION THREAD

**Discussion by virtadpt** [Part-DB/Part-DB-server#270] [General]
As part of regular webapp maintenance, I like to trim debugging logs and suchlike to keep them of a manageable size.  `var/prod.log` isn't too bad for this.  However, it looks like the system event log is stored in the database alongside everything else.  Is there a way to truncate the system event log so the DB won't get too big?  I went through the output of `bin/console` a couple of times and, aside from finding on-disk cache management stuff I didn't see anything about truncating the system event table.  Is there a setting that I'm missing someplace?

**Comment by jbtronics** [comment]
I would not recommend to clear the log, as it is used to maintain the change history of data. So if you remove these entries, you will loose the entire history of parts (and another elements).
Maybe you could delete some older log entries about login/logout of users and so.

Currently there is no builtin possibility to delete log entries (event though there is a permission for "delete logs"). I am  currently working on some improvements of the log content, it will allow administrators with the permission to delete log entries (mostly for privacy reasons or similar).
